web/package.json

```diff
@@ -14,12 +14,14 @@
     "fix": "prettier --write src/ && eslint --fix src/"
   },
   "devDependencies": {
-    "@parcel/transformer-sass": "^2.2.0",
+    "@parcel/transformer-sass": "2.2.0",
     "eslint-config-google": "0.14.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-prettier": "4.0.0",
     "parcel": "^2.0.0",
-    "prettier": "2.4.1"
+    "prettier": "2.4.1",
```

Comment:
Since we're linting things, can we add a check to `npm run lint` that makes sure all things have been prettified?

Reply:
I had added the test in `workflows/tests.yml` but this is better. Thanks for mentioning that.

Reply:
I'd like to see us incorporate prettier into the `lint` run script. But that can wait for a future PR.

Reply:
Yeah, surely :)